src/tests/setupTests.ts

```diff
@@ -0,0 +1,49 @@
+import '@testing-library/jest-dom'
+
+jest.mock('i18next', () => ({
+  t: (key: string) => key,
+  changeLanguage: jest.fn(() => Promise.resolve()),
+  language: 'en',
+  languages: ['en'],
+  isInitialized: true,
+  exists: jest.fn(() => true),
+  getFixedT: jest.fn(() => (key: string) => key),
+  hasResourceBundle: jest.fn(() => true),
+  loadNamespaces: jest.fn(() => Promise.resolve()),
+  loadLanguages: jest.fn(() => Promise.resolve()),
+  off: jest.fn(),
+  on: jest.fn(),
+  emit: jest.fn(),
+  store: {},
+  services: {},
+  options: {},
+}))
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key) => key,
+    i18n: {
+      changeLanguage: jest.fn(() => Promise.resolve()),
+      language: 'en',
+      languages: ['en'],
+      isInitialized: true,
+      exists: jest.fn(() => true),
+      getFixedT: jest.fn(() => (key) => key),
+      hasResourceBundle: jest.fn(() => true),
+      loadNamespaces: jest.fn(() => Promise.resolve()),
+      loadLanguages: jest.fn(() => Promise.resolve()),
+      off: jest.fn(),
+      on: jest.fn(),
+      emit: jest.fn(),
+      store: {},
+      services: {},
+      options: {},
+    },
+  }),
+  initReactI18next: {
+    type: '3rdParty',
+    init: jest.fn(),
```

Comment:
I just discovered that there are numerous lint errors generated from these lines in the setupTest. Could you also take a look?